test/Rings/AbelianClosure.jl

```diff
@@ -86,6 +86,17 @@
     @test_throws Hecke.NotImplemented Oscar.AbelianClosure.coerce_down(Hecke.rationals_as_number_field()[1], 1, z(2))
   end
 
+  @testset "Conversion" begin
+    K, z = abelian_closure(QQ)
+    x  = z(5)
+    y = ZZ(x^5)
+    @test y isa fmpz
+    @test y == 1
+    y = QQ(x^5)
+    @test y isa fmpq
+    @test y == 1
+  end
```

Comment:
Perhaps also verify what happens with invalid input?
```suggestion

    @test_throws ErrorException ZZ(x)
    @test_throws ErrorException QQ(x)
  end
```

Reply:
Yes. Thanks.